Jenkinsfile

```
@@ -198,22 +239,21 @@ pipeline {
                                    inst_repos: "daos@master"
                     runTest script: '''NODE=${NODELIST%%,*}
                                        ssh $SSH_KEY_ARGS jenkins@$NODE "set -ex
-                                       yum --disablerepo=\\* --enablerepo=build\\* makecache"''',
+                                       yum --disablerepo=\\* \
+                                         --enablerepo=build\\* makecache"''',
```

```suggestion
                                            --enablerepo=build\\* makecache"''',
```
to line up with the arg. on the previous line.

---

Will be in next push